### ISSUE TITLE
Understanding WCAG 1.4.5 references 1.4.4, should be 1.4.9

### ISSUE THREAD

**Issue by kirabug** [w3c/wcag#1477]
Navigation:
1. Go to https://www.w3.org/WAI/WCAG21/Understanding/images-of-text.html 
2. Scroll to the bottom of the Intent section, just above the Benefits header. 
3. Observe the last paragraph, "See also 1.4.4: Images of Text (No Exception)."

Issue:
That paragraph links to 1.4.9 Images of Text (No Exception), but is titled "1.4.4". 

Note: There's another reference to 1.4.4 in the 2nd paragraph, but I can't tell if that's intentionally referencing 1.4.4 Resize Text or whether it's also a typo.

**Comment by scottaohara** [comment]
actually made a PR for this the other day, #1466

**Comment by alastc** [comment]
Thanks both, as it's small editorial change I've merged that straight in. It might take a little while to go live though.
